src/r_skydefs.c

```diff
@@ -14,15 +15,20 @@
 #include "r_skydefs.h"
 
 #include "doomdef.h"
+#include "doomstat.h"
 #include "doomtype.h"
 #include "i_printf.h"
 #include "m_array.h"
 #include "m_fixed.h"
 #include "m_json.h"
 #include "m_misc.h"
 #include "r_data.h"
+#include "r_sky.h"
 
-static boolean ParseFire(json_t *json, fire_t *out)
+flatmap_t *skyflats;
+sky_t     *skies;
```

Comment:
We don't need those anymore.

Reply:
I didn't fetch the new code since yesterday, thanks!